cmd/cluster/core/destroy.go

```diff
@@ -166,7 +192,13 @@ func DestroyCluster(ctx context.Context, hostedCluster *hyperv1.HostedCluster, o
 		}
 	}
 
-	o.Log.Info("Successfully destroyed cluster and infrastructure", "namespace", o.Namespace, "name", o.Name, "infraID", o.InfraID)
+	o.Log.Info("Successfully destroyed cluster", "namespace", o.Namespace, "name", o.Name, "infraID", o.InfraID)
+	if hciFound {
+		o.Log.Info(fmt.Sprintf("You can now delete the HostedClusterInfrastructure resource (Namespace/Name) %s/%s\nUse the command \"kubectl -n %s delete hci %s\"",
```

Comment:
nit: please use client.ObjectKeyFromObject(your-object).String() to render resources and %q for strings to avoid scaping character for better readability

Reply:
no longer applies.